.github/workflows/comment_on_pr.yaml

```diff
@@ -0,0 +1,79 @@
+---
+name: Comment output on PR
+
+on:
+  workflow_run:
+    workflows: ["Vanagon Component Diff"]
+    types:
+      - completed
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - name: 'Download artifacts'
+        uses: actions/github-script@v3.1.0
+        with:
+          script: |
+            var artifacts = await github.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: ${{github.event.workflow_run.id }},
+            });
+            var matchArtifact = artifacts.data.artifacts.filter((artifact) => {
+              return artifact.name == "artifacts"
+            })[0];
+            var download = await github.actions.downloadArtifact({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               artifact_id: matchArtifact.id,
+               archive_format: 'zip',
+            });
+            var fs = require('fs');
+            fs.writeFileSync('${{github.workspace}}/artifacts.zip', Buffer.from(download.data));
+            fs.writeFileSync('./artifactid', matchArtifact.id)
+
+      - name: 'Unzip artifacts'      
```

Comment:
Can we use https://github.com/actions/download-artifact ?

Reply:
It seems that this action is useful for using in the same workflow and doesn't have a parameter to change workflow id from where to retrieve artifacts.